dev-debug/bpftrace/files/bpftrace-0.21.3-cstdint.patch

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Drop this newline in future?

Reply:
It's already in upstream so that future won't come. :)

Reply:
Ah, you meant in general. Sure, will do that.